### DISCUSSION TITLE
the default value  of bool column does'not work when create

### DISCUSSION THREAD

**Discussion by maxiaojunwolf** [aminalaee/sqladmin#754] [Q&A]
class ViewConfig(Base):

    __tablename__ = "config"

    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
    pk_columns: Mapped[str] = mapped_column(String(128), default="id")
    is_async: Mapped[bool] = mapped_column(Boolean, default=True)

![image](https://github.com/aminalaee/sqladmin/assets/34365493/e544dd3c-ce96-43d7-86d6-4654a518252d)

fastapi: 0.110.2
sqladmin: 0.16.1

**Comment by alserious** [comment]
same problem

**Comment by mmzeynalli** [comment]
Is this still a problem?
